src/registries/menus/row_menu_registry.ts

```diff
@@ -67,11 +72,8 @@ rowMenuRegistry
     action: ACTIONS.HIDE_ROWS_ACTION,
     isVisible: (env: SpreadsheetChildEnv) => {
       const sheetId = env.model.getters.getActiveSheetId();
-      const hiddenRows = env.model.getters.getHiddenRowsGroups(sheetId).flat();
-      return (
-        env.model.getters.getNumberRows(sheetId) >
-        hiddenRows.length + env.model.getters.getElementsFromSelection("ROW").length
-      );
+      const visibleRows = env.model.getters.getAllVisibleHeaders(sheetId, "ROW").length;
+      return visibleRows > env.model.getters.getElementsFromSelection("ROW").length;
```

Comment:
Same question about the case were we have more hidden columns than visible columns (e.g. B to C hidden) and we select A to D. Will `getElementsFromSelection` only contains `[A, D]` or ´[A, B, C, D]` ?